### ISSUE TITLE
Improve performance of defaultTokenizerNormalizer

### ISSUE THREAD

**Issue by scossin** [scossin/IAMsystem#9]
The class TokenizerNormalizer is a very important one in the pipeline. It handles the tokenization and the normalization of a document and the terms of a terminology.Users can customize the tokenization and normalization process.
After tokenization (out: String[] tokens] and normalization(out: String normalized) the class computes the position (start,end) of each token (int[][] tokenStartEndInSentence) by iterating over each character. 

This process is not efficient for the defaultTokenizerNormalizer which uses a whiteSpaceTokenizer: the best way is to compute the position (start,end) during the normalization / tokenization ; avoiding iterating twice on the characters of a document.

**Comment by scossin** [comment]
fix 3659910cfa403a405fe935f7ed9dcf786c36e795

The new *SimpleTokenizerNormalizer* class is about twice as fast at tokenizing and normalizing a document with whitespace and keeping alphanumeric character

```java
ITokenizerNormalizer tn2 = new SimpleTokenizerNormalizer();
TokenizerNormalizer tn = TokenizerNormalizer.getDefaultTokenizerNormalizer();
long startTimeAll = System.nanoTime();
for (int i = 0; i<10000; i++) {
	tn.tokenizeNormalize(document);
}
long endTimeAll = System.nanoTime();
long totalTime = endTimeAll - startTimeAll;
System.out.println(totalTime / 1000000);
```

**Comment by scossin** [comment]
Fixed release in version 1.3.1 e737d1949f38d875abc6ffcf4d1bee8fd5c97304

**Comment by scossin** [comment]
Issue not relevant anymore since publication of the latest version v2.1.0
